OK same error here;

```
=== Failed
=== FAIL: pkg/e2e TestPublishChecks/detect_sensitive_data (0.08s)
    publish_test.go:152: assertion failed: expression is false: strings.Contains(output, "JSON Web Token\n\"\": ***"+"eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw"): you are about to publish sensitive data within your OCI artifact.
        please double check that you are not leaking sensitive data
        AWS Client ID
        "services.serviceA.environment.AWS_ACCESS_KEY_ID": A3TX1234567890ABCDEF
        AWS Secret Key
        "services.serviceA.environment.AWS_SECRET_ACCESS_KEY": aws"12345+67890/abcdefghijklm+NOPQRSTUVWXYZ+"
        Keyword Detector
        "secrets.mysecret.file": /home/runner/work/compose/compose/pkg/e2e/fixtures/publish/secret.txt
        Github authentication
        "GITHUB_TOKEN": ***
        Private Key
        "": -----BEGIN DSA PRIVATE KEY-----
        wxyz+ABC=
        -----END DSA PRIVATE KEY-----
        Are you ok to publish these sensitive data?
        
    --- FAIL: TestPublishChecks/detect_sensitive_data (0.08s)
```

--- a/go.mod
+++ b/go.mod
@@ -81,7 +81,7 @@ require (
 	github.com/go-logr/logr v1.4.3 // indirect
 	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/gofrs/flock v0.13.0 // indirect
-	github.com/golang-jwt/jwt/v5 v5.3.0 // indirect
+	github.com/golang-jwt/jwt/v5 v5.3.1 // indirect
 	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510 // indirect
 	github.com/gorilla/mux v1.8.1 // indirect

